packages/reassure-compare/src/compare.ts

```diff
@@ -56,33 +65,32 @@ export async function compare({
     process.exit(1);
   }
 
-  const current = await loadFile(currentFile);
+  const currentMetadata = await loadMetadataFromFile(currentFile);
```

Comment:
Maybe we could tweak the original method to return structured data:

```suggestion
  const { metadata, results } = await loadFile(currentFile);
```

Reply:
The reason I had made a separate method for these was because I saw a TODO around name tag validation.

I thought if we could have separate methods for each validation then it would make more sense so that I could then refactor out `readFileAndParseDataToFormat` to make some validations based on tagnames

Reply:
But I get your point, I will re-add the older method and make it load both performance results and metadata

Reply:
Have addressed this in latest commits.